rapids_pytest_benchmark/rapids_pytest_benchmark/plugin.py

```diff
@@ -370,10 +378,11 @@ def __init__(self, benchmarkSession):
         # think?)
         origColumns = self.columns
         self.columns = []
-        for c in ["min", "max", "mean", "stddev", "median", "iqr", "outliers", "ops", "gpu_mem", "rounds", "gpu_rounds", "iterations"]:
+        for c in ["min", "max", "mean", "stddev", "median", "iqr", "outliers", "ops", "gpu_mem", "gpu_leaked_mem", "rounds", "gpu_rounds", "iterations"]:
             # Always add gpu_mem (for now), and only add gpu_rounds if rounds was requested.
```

Comment:
minor: might want to update this comment since it's also always adding `gpu_leaked_mem`